TripalCultivate/TripalCultivate
# Creating Content Types

Issue #4 

## Description

This PR will create content types specific to Tripal cultivate. Currently it is focused on experiments and sample types. I'm putting this up as a draft PR to let everyone know where I am and keep me accountable 🤪 

## Testing

1. Terms are imported on install so check that they exist in chado.cvterm after creating a fresh docker on this branch.
2. This PR creates two "Tripal type collections" so go to Tripal Admin toolbar > Page Structure and click the "Import type collection" button. 
3. Select the two for Tripal Cultivate and click import.
4. Run the Tripal job and ensure there are no errors.
5. Go back to the page structure page and confirm the new content types are present!
